CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to [bpmnlint](https://github.com/bpmn-io/bpmnlint) are docum
 
 ___Note:__ Yet to be released changes appear here._
 
+* `FEAT`: add `no-missing-di` rule ([#55](https://github.com/bpmn-io/bpmnlint/issues/55))
```

Comment:
Proposal: Rename rule to `missing-bpmndi` or `no-bpmndi`? No missing is _doppeltgemoppelt_, kind of.

Reply:
+1